src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -516,10 +525,14 @@ class CosmasGoogleCloudService(
         } else {
             FileCemetery.parseFrom(cemeteryBytes.getContent()).toBuilder()
         }
+        val curTime = clock.millis()
+        val ttl = getTTL(request.info)
+        val tombs = cemetery.cemeteryList.toMutableList()
+        tombs.removeIf { it.removalTimestamp + ttl > curTime }
```

Comment:
Did you really mean `>` here?

Reply:
It was a mistake in deprecated `deleteFile`. I combined `deleteFiles` with `deleteFile`, so they both use same code.